mdflynn/animetrics
# Feature/main page

## Is this a fix or a feature?
This is a feature and a fix

## What is the change?
No change this PR.

## What does it fix?
Corrects an issue where the endpoints were not updating on Link click

## Where should the reviewer start?
Review App.js, MainPage.js, index.js, EpisodeDisplay.js in full

## How should this be tested?
Run `npm test` and navigate to localhost:3000 to see current site functionality.

## Relevant Issues
#2 #3
